gamemodes/terrortown/gamemode/client/vgui/cl_sb_team.lua

```diff
@@ -123,51 +131,62 @@ function PANEL:HasRows()
 	return self.rowcount > 0
 end
 
-function PANEL:UpdateSortCache()
-	self.rows_sorted = {}
-
-	for _, row in pairs(self.rows) do
-		table.insert(self.rows_sorted, row)
+local function SortFunc(rowa, rowb)
+	if not IsValid(rowa) or not IsValid(rowb) then
+		return false
 	end
 
-	local cv_ttt_scoreboard_sorting = GetConVar("ttt_scoreboard_sorting")
-	local cv_ttt_scoreboard_ascending = GetConVar("ttt_scoreboard_ascending")
-
-	table.sort(self.rows_sorted, function(rowa, rowb)
-		local plya = rowa:GetPlayer()
-		local plyb = rowb:GetPlayer()
-
-		if not IsValid(plya) then
-			return false
-		end
+	local plya = rowa:GetPlayer()
+	local plyb = rowb:GetPlayer()
 
-		if not IsValid(plyb) then
-			return true
-		end
+	if not IsValid(plya) then
+		return false
+	end
 
+	if IsValid(plyb) then
```

Comment:
Why not checking IsValid(plyb) above as well and just return true if it isn't?

Reply:
Because then the sorting error pops up

Reply:
Really? This should result in exactly the same behavior as before

```if IsValid(plyb) then .... end return true ``` as it is currently vs my appraoch: 
```if not IsValid(plyb) then return true end ... ```

Reply:
The sorting function does strange things when you try to return true midway through. But feel free to improve it I gues